Example/Controllers/Extensions/ViewController+MiniAppUserInfoProtocol.swift

```diff
@@ -23,11 +23,17 @@ extension ViewController {
         return contactList
     }
 
-    func getAccessToken(miniAppId: String, completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void) {
-        guard let tokenInfo = getTokenInfo() else {
-            completionHandler(.success(.init(accessToken: "ACCESS_TOKEN", expirationDate: Date())))
-            return
+    func getAccessToken(miniAppId: String,
+                        scopes: MASDKAccessTokenScopes?,
+                        completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void) {
+        var resultToken = "ACCESS_TOKEN"
+        var resultDate = Date()
+        let resultScopes = scopes
+
+        if let tokenInfo = getTokenInfo() {
+           resultToken = tokenInfo.tokenString
+           resultDate = tokenInfo.expiryDate
         }
-        completionHandler(.success(.init(accessToken: tokenInfo.tokenString, expirationDate: tokenInfo.expiryDate)))
+        completionHandler(.success(.init(accessToken: resultToken, expirationDate: resultDate, scopes: resultScopes)))
```

Comment:
Looks like the JS Bridge code needs to be updated too. The Mini App currently isn't receiving the "scopes" because it's not parsed in the bridge code.